ui/js/controllers/jobs.js

```diff
@@ -109,6 +109,48 @@ treeherderApp.controller('ResultSetCtrl', [
             $rootScope.selectedJob = job;
         };
 
+        $rootScope.$on(thEvents.applyNewJobs, function (ev, resultSetId) {
+            if ($scope.resultset.id === resultSetId) {
+                $scope.getDecisionTaskVisibility();
+            }
+        });
+
+        $scope.getDecisionTaskVisibility = function () {
+            let dtId = ThResultSetStore.getGeckoDecisionTaskId($scope.repoName, $scope.resultset.id);
+            if (dtId.$$state.value === 'No decision task') {
```

Comment:
I'm now getting new console complaints `Possibly unhandled rejection: No decision task` that I didn't see in the originally merged branch. Still something to fix here. Maybe Angular changed to 1.6 between my original branch work and this new PR.

Various kb's on the topic
http://www.codelord.net/2017/08/20/angular-1-dot-6-s-possibly-unhandled-rejection-errors/

Reply:
I'll put up a supplementary commit with the change. I'm just writing the error to the console in the WIP commit, I assume we'd perhaps write the error condition to ThLog.